ApplicationInsights/JavaScriptSDK.Interfaces/IPageViewTelemetry.ts

```diff
@@ -6,29 +6,29 @@ export interface IPageViewTelemetry {
      * name The string you used as the name in startTrackPage. Defaults to the document title.
      */
```

Comment:
Here are the properties 
IPageViewTelemetry {
    name?: string;
    id: string;
    uri?: string;
    refUri?: string;
    pageType?: string;
    duration?: number;
    isLoggedIn?: boolean;
    ver?: number;
    pageTags?: { [key: string]: any };
}

Reply:
For now in Channel, we need to put refUri in part C until kusto pipeline is open for changes.

Reply:
Sorry refUri, pageType, isLoggedIn, pageTags should all go to part C in channel until kusto pipeline opens for changes.

Reply:
also, ver does not need to be exposed to customer here, so no need to add

Reply:
do not put isLoggedIn, id, ver

---
In reply to: [204725364](https://github.com/Microsoft/ApplicationInsights-JS/pull/635#discussion_r204725364) [](ancestors = 204725364)

Reply:
Put the strcuture as in in baseData and let the channel put pageTags in part C

---
In reply to: [204865082](https://github.com/Microsoft/ApplicationInsights-JS/pull/635#discussion_r204865082) [](ancestors = 204865082,204725364)